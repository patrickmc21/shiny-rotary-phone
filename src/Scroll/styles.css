.scroll {
  float: left;
  height: 800px;
  width: 40vw;
}