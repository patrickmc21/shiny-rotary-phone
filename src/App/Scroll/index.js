import React from 'react';
import PropTypes from 'prop-types';
import './styles.css';

const Scroll = () => {
  return (
    <div>I'm the Scroll</div>
  )
}

export default Scroll;