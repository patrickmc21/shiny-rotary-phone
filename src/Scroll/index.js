import React from 'react';
import PropTypes from 'prop-types';
import './styles.css';

const Scroll = ({scrollData}) => {
  return (
    <aside className='scroll'>
      <article>
        <p>{scrollData.opening_crawl}</p>
        <h4>{scrollData.title}</h4>
        <time>{scrollData.release_date}</time>
      </article>
    </aside>
  )
}

Scroll.propTypes = {
  scrollData: PropTypes.string
}

export default Scroll;