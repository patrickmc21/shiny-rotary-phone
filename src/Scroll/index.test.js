import React from 'react';
import { shallow, mount } from 'enzyme';
import Scroll from './index.js';

describe('Scroll', () => {

  it('should match the snapshot', () => {
    const wrapper = shallow(<Scroll />);
    expect(wrapper).toMatchSnapshot();
  })
})