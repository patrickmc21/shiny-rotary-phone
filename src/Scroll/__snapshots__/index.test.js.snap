// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scroll should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Scroll />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <article />,
      "className": "scroll",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": undefined,
      },
      "ref": null,
      "rendered": null,
      "type": "article",
    },
    "type": "aside",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <article />,
        "className": "scroll",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": "article",
      },
      "type": "aside",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
